modules/cnf-managing-ha-nrop-scheduler.adoc

```diff
@@ -0,0 +1,128 @@
+= Managing high availability (HA) for the NUMA-aware scheduler
+
+The NUMA Resources Operator manages the high availability (HA) of the topology-aware secondary scheduler based on how the `spec.replicas` field is configured in the `NUMAResourcesScheduler` Custom Resource (CR). Control scheduler behavior by using one of three available options.
+
+--
+:FeatureName: Managing high availability
+include::snippets/technology-preview.adoc[]
+--
+
+[discrete]
+[id="default-ha-mode_{context}"]
+== Default high-availability mode
+
+If the `spec.replicas` field is not specified in the `NUMAResourcesScheduler` custom resource (CR), the NUMA Resources Operator automatically enables HA mode. In this mode, the Operator creates one scheduler replica for each control-plane node in the cluster to ensure redundancy.
+
+The following manifest demonstrates this default behavior. Omit the replicas field to enable automatic replica detection. 
+
+[source,yaml,subs="attributes+"]
+----
+apiVersion: nodetopology.openshift.io/v1
+kind: NUMAResourcesScheduler
+metadata:
+  name: example-auto-ha
+spec:
+  imageSpec: 'registry.redhat.io/openshift4/noderesourcetopology-scheduler-rhel9:v{product-version}'
+  # The 'replicas' field is not included, enabling auto-detection.
+----
+
+[discrete]
+[id="customizing-replicas_{context}"]
+== Customizing replicas
+
+You can explicitly set a specific number of scheduler replicas by defining the `spec.replicas` field. This action overrides the default HA behavior. The controller then ensures the number of running scheduler pods matches the specified value.
+
+[source,yaml,subs="attributes+"]
+----
+apiVersion: nodetopology.openshift.io/v1
+kind: NUMAResourcesScheduler
+metadata:
+  name: example-custom
+spec:
+  imageSpec: 'registry.redhat.io/openshift4/noderesourcetopology-scheduler-rhel9:v{product-version}'
+  replicas: 2
+----
+
+[discrete]
+[id="disabling-numa-aware-scheduling_{context}"]
+== Disabling NUMA-aware scheduling
+
+To disable the scheduler, set the `spec.replicas` field to `0`. The NUMA Resources Operator then terminates all running scheduler pods and prevents new ones from starting.
+
+The following manifest disables the scheduler:
+
+[source,yaml,subs="attributes+"]
+----
+apiVersion: nodetopology.openshift.io/v1
+kind: NUMAResourcesScheduler
+metadata:
+  name: example-custom
+spec:
+  imageSpec: 'registry.redhat.io/openshift4/noderesourcetopology-scheduler-rhel9:v{product-version}'
+  replicas: 0
+----
+
+.Verification 
+
+. Run the following command with a label selector to list only the scheduler pods:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-numaresources -l app=secondary-scheduler
+----
++
+.Expected Output
+
+If you used the **default HA mode**, you should see that the number of pods is equal to the number of control-plane nodes. For a standard HA {product-title} cluster, which typically has three control plane nodes, you would expect to see three pods, for example:
```

Comment:
ditto, lets mentioned it's capped at 3 replicas in the auto-detection mode.

Reply:
Also on HyperShift mode the replicas (scheduler pods) are running on the worker nodes on the hosted-cluster.
This is trivial for HyperShift because the operator itself is running on the hosted-cluster as well.